README.md

```diff
@@ -109,20 +109,26 @@ To compile gqrx from source you need the following dependencies:
     - Network
     - Widgets
     - Svg (runtime-only)
-- cmake version >= 3.2.0
+- cmake version >= 3.13.0
 
 Gqrx can be compiled from within Qt Creator or in a terminal:
 
 For command line builds:
 <pre>
 $ git clone https://github.com/gqrx-sdr/gqrx.git gqrx.git
 $ cd gqrx.git
-$ mkdir build
-$ cd build
-$ cmake ..
-$ make
+gqrx.git$ mkdir build
+gqrx.git$ cmake -S . -B build
+gqrx.git$ cmake --build build --parallel
+gqrx.git$ cmake --install build --prefix artifacts/
```

Comment:
Why install to `artifacts`? We should probably recommend installing to the system, adding `sudo`.

Reply:
There are a lot of discussions why "don't use make install", for [example](https://www.reddit.com/r/linux/comments/16iizeh/why_sudo_make_install/). The main point: usually gqrx is working under some distro like Ubuntu, not on embedded systems. For embedded systems `sudo cmake --install build` is ok but for systems with packet manager (like apt) such code will break system packages...
As a compromise we can add to readme.md `sudo cmake --install build` way as optional and dangerous. Do you agree?